framework/include/base/ResidualObject.h

```diff
@@ -67,7 +66,7 @@ class ResidualObject : public MooseObject,
    * Computes Jacobian block of the variable of this object with respect to a field variable
    * @param jvar The field variable
```

Comment:
It will be nice to add some docs here to explain why we should to pass variable number here. Note this line of doxygen comment is not accurate anymore.